### DISCUSSION TITLE
Screenshots of the web interface

### DISCUSSION THREAD

**Discussion by fplanque** [Hypfer/Valetudo#1976] [Q&A / Support] [answered]
Before I void my warranty and install Valetudo, can I see the valetudo web interface somewhere?

All the screenshots I find seem to be for the android app. I don't have an android device.

Could someone please point me to some please where I can see what the web interface looks like?

Thank you so much!

**Comment by rimrul** [comment, marked as answer]
The screenshots in https://github.com/Hypfer/Valetudo/blob/master/README.md are the webinterface. The only android app I know of is https://github.com/Hypfer/valetudo-companion, but that's probably not what you're talking about.

**Comment by fplanque** [comment]
Ok, thank you for that. 

I assumed those were app screenshots because of the vertical screen ratio and no URL address bar visible.
